### ISSUE TITLE
Error during first start: bash: -c: line 0: unexpected EOF while looking for matching `'' bash: -c: line 1: syntax error: unexpected end of file

### ISSUE THREAD

**Issue by Grovespaz** [kubernetes/minikube#1194]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Bug report

**Minikube version** (use `minikube version`): v0.16.0

**Environment**:
- **OS** (e.g. from /etc/os-release): Windows 10 version 1607 (build 14393.693)
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): hyperv
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): minikube-v1.0.6.iso
- **Install tools**: ?
- **Others**: kubectl v1.5.3

**What happened**:
When running ```minikube start``` for the first time (```minikube start --vm-driver=hyperv``` to be accurate), the terminal fills up with this message repeated multiple times:
```
Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQGmZb9U5N8qBCCZ7xg53WcDANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI3MDgxNjAwWhcNMjAwMjEyMDgxNjAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDHQC/cehYoiR8sCNYpEfg1s1rY48a7Df9i61zeGINBTz7c7CaXn5SITFuy
KvN6W0RVyKx7p1gT0yrgnFhF4DnmoQM3F/CaIYsDGp19LygGkfsp2KZlRkVXvdq1
HTSS+pbh/S25wtSI1WdWyxStyUXTFxS+qRzA6tw+6tXK5LZhpCFz1Mq3Rv6se7ce
Hdw++zyqTtu+x4NIUHY9q/18+eESsfUXOhkB3tMs2ZqvEEVqW+IDVvRI4GM6YlKU
Si0QwBxvDR9yhnxAtOCfKzPQmvFi/zMZhy8NIltEdS/IwalmqpFplHqimBlgb6xM
takjPEIj0I8Ji+qLlo9nbM02EtZxAgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQCW6o8i4V/rk/gK3Wtt
GYfvIji6XM7TCr00tqhzbNkWOJKVIybn0mOy8+xb4Ny30JqX6P9BT3ygqHDr3GR8
VlXiF9bFuUY2ZYv4AZuu35CEkJ5irR6VPaaB2bUmknYIZwE1c5TsdXlz/N/kZFTx
ndi1bhwKPIftH/FlmJRHdIbNiWLkFP2v5tC7fsmKQI85ZLcaVDN3VYPgnPQ3ggj8
8PdMtrHt43axu/ngkfOGERharcMANq5zmpVhM6Iy8gsv+6VH5G3YGCA9+4RkNVRV
0j30Z45+lrdW3YD21cd4VpvWIsd7X39NfzuIhq6kBLYl07XHVyQPJroIzlCuCjo0
hGmN
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file
```
Repeating the command yields the same result.
Running ```minikube status``` results in:
```
E0227 13:52:33.938528   12204 status.go:57] Error getting machine status: Error: Unrecognize output from GetLocalkubeStatus:
E0227 13:52:33.951492   12204 util.go:134] Error formatting error message: Error msg with no stack trace cannot be reported
```

**What you expected to happen**:
minikube start not to fail

**How to reproduce it** (as minimally and precisely as possible):
Run ```minikube start --vm-driver=hyperv``` from an elevated command prompt on a Windows 10 machine.

**Comment by aaron-prindle** [comment]
@Grovespaz Could you try running the command with Powershell?  Trying to isolate if the issue has to do with the shell used.

**Comment by Grovespaz** [comment]
Running the command in Powershell after removing the VM from HyperV and removing ~/.minikube yields the same result.

Just in case, here's the complete output:
```
PS C:\WINDOWS\system32> minikube start --vm-driver=hyperv
Starting local Kubernetes cluster...
Downloading Minikube ISO
 84.07 MB / 84.07 MB [==============================================] 100.00% 0s
E0228 08:46:50.000944   25944 start.go:107] Error starting host: Error creating host: Error creating machine: Error running provisioning: Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file

.

 Retrying.
E0228 08:46:50.004731   25944 start.go:113] Error starting host:  Error creating host: Error creating machine: Error running provisioning: Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


Temporary Error: Something went wrong running an SSH command!
command : printf '%s' '-----BEGIN CERTIFICATE-----
MIICzzCCAbegAwIBAgIQLPfGTzw6MefX+TL58P2g6DANBgkqhkiG9w0BAQsFADAR
MQ8wDQYDVQQKEwZnb3ZlcnQwHhcNMTcwMjI4MDczOTAwWhcNMjAwMjEzMDczOTAw
WjARMQ8wDQYDVQQKEwZnb3ZlcnQwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEK
AoIBAQDZR+asZCSt2zNMGDGSnqEMBpDKY7exTWdVWFBw3i1WJHGYspalzftfFUoo
Oeh1Za324OEcOzAxRqLAlR8jQKVBB51g8e4gS6xAVzle/MF9b9UHQYMAw1YM8GDi
HVQcn61AEEceDRNpoaG5rEUD5oztYK2FpwQRpOTS1ZVbmU/pJbS7QmAPmYaTokp+
7ahmS3MDmADfsPg26KWhC9Tj75AJvQ+97tMA5JtIV8g2m32Avfhf0zV5KJHOQelE
n3X6XGm+6J1M8QBzmDqZx/eafAt2sm2bHq5OEEWLQBQv/cF8mrvOm8dPK8yQ+yLc
rzs85Kkv1/T4zZpuiNpMmGgeqxh5AgMBAAGjIzAhMA4GA1UdDwEB/wQEAwICrDAP
BgNVHRMBAf8EBTADAQH/MA0GCSqGSIb3DQEBCwUAA4IBAQDSymN8QT6iXX04rmJ6
+Q6UhVN/JIDo1GurYZxpoFNUGlOFTtlCdprTUcjBc4y6eCwqxm7RPYlvwgyF4Ier
RA9kzLdyvFhqHRQshLinTbnfkGRdv0qCyw8NG/QW89CyPeB+IVZ4dQLGqyEVpczc
RK4/Y4ZqrITmwXtMwGaBS9b21xcBaDtSN7L9ytvJXOypnchJPUBQrfZPJUvENVqh
qQfKhOfr7mj4o+U5VR5AheCJWUc9gRSqj6ylUGivwSryMvYxF60cCHl1qN1U6Ap6
m6U5pdT32MJDZkk0fW5NLV9j9L9RlskNMy0vDcHl5a/PvxEVXkLux+e3gwjlGuFz
erSn
-----END CERTIFICATE-----
' | sudo tee /etc/docker/ca.pem
err     : exit status 1
output  : bash: -c: line 0: unexpected EOF while looking for matching `''
bash: -c: line 1: syntax error: unexpected end of file


================================================================================
An error has occurred. Would you like to opt in to sending anonymized crash
information to minikube to help prevent future errors?
To opt out of these messages, run the command:
        minikube config set WantReportErrorPrompt false
================================================================================
Please enter your response [Y/n]:
```

**Comment by aaron-prindle** [comment]
Can you try running this with libmachine logging enabled `--v=7`.  Also make sure that you are designating a hyperv virtual switch as the driver will select one randomly if you don't `--hyperv-virtual-switch=YourVSwitch` (right now if might be trying to use DockerNat if you have Docker For Windows installed): 
`minikube start --vm-driver=hyperv --v=7 --hyperv-virtual-switch=YourVSwitch`

**Comment by Grovespaz** [comment]
I apologize for the late reply. Thanks for the guidance! The VM was indeed connected to the DockerNat switch but solving this didn't solve the original error.

After going through the verbose log, I noticed a call to a stray file called ssh.bat helpfully left in my path by the chromium build tools :) After removing this from my path, minikube worked as expected.

Thank you for your time and apologies for the noise.
As a closing thought, I saw that minikube has a built-in go fallback for SSH. What would the downsides be to always using this as to not depend on (possibly faulty) user configuration?
